components/dashboard/queries.ts

```diff
@@ -26,7 +26,22 @@ export const adminPanelQuery = gql`
       duplicatedAccounts {
         totalCount
       }
-      pendingExpenses: expenses(status: PENDING, direction: RECEIVED, includeChildrenExpenses: true, limit: 0) {
+      pendingExpenses: expenses(
```

Comment:
Why do we load these counts here? I think we're downplaying the performance impacts of counting expenses and contributions when loading the dashboard.

Reply:
Its currently part of the Overview page.

![image](https://github.com/user-attachments/assets/fb4d1049-9739-4f88-ac27-63a25f5fad8a)

When we split grants into a separate tool, we need to differentiate the count on the overview as well.

Reply:
Sure, not a problem to be solved here.
Ideally overview loads its own information, so we can load the menu earlier in case the user wants to navigate to a different view when loading the dashboard. cc @gustavlrsn